DcCore/DcCore/DC/events.swift

```diff
@@ -1,7 +1,11 @@
 import UIKit
 import UserNotifications
 
-public let eventMsgsChangedReadDeliveredFailed = Notification.Name(rawValue: "eventMsgsChangedReadDeliveredFailed")
+extension Notification.Name {
+    public static let messageChanged = Notification.Name(rawValue: "eventMsgsChanged")
```

Comment:
it's plural [as multiple messages may be affected](https://c.delta.chat/group__DC__EVENT.html#ga0f52cdaad70dd24f7540abda6193cc2d). we should stay close to the original meaning here - so either `msgsChanged or messagesChanged`.

(read/delivered/failed/reactionsChanged otoh affect only a single message and singular is fine therefore)

Reply:
🫨😵‍💫😅👍